### ISSUE TITLE
Cross entropy loss gradient

### ISSUE THREAD

**Issue by victorhcm** [tuan3w/cs231n#1]
Hi @tuan3w, 

I have recently started doing the course and I'm currently trying to compute the gradient of the cross entropy loss for the Softmax function. I was able to find the gradient w.r.t. [$f_{y_i}$](https://github.com/tuan3w/cs231n/blob/master/assignment1/cs231n/classifiers/softmax.py#L39), but I'm having trouble to compute the gradient with regarding [$f_{j}$](https://github.com/tuan3w/cs231n/blob/master/assignment1/cs231n/classifiers/softmax.py#L37). In your gradient, you have found $\frac{e^{f_j}}{\sum_j{e^{f_j}}} * x_i$, while I'm finding only $x_i$.

How have you done it? :)

Thanks!

**Comment by tuan3w** [comment]
Hi @victorhcm ,
It takes a lot of time for me to compute the derivative  of softmax function :D. The final formulation of it you can found at http://cs231n.github.io/neural-networks-case-study/. You can check it by yourself.

**Comment by victorhcm** [comment]
Thank you, @tuan3w! I didn't see it there :D
